src/plugin_system/plugin_system.cpp

```diff
@@ -15,15 +15,20 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
+#include <dlfcn.h>
```

Comment:
doublecheck that this is available on windows platforms (ci should be enough)

Reply:
This has been delayed. See https://github.com/sillydan1/aaltitoad/issues/45 for details